build/python_metadata.bzl

```diff
@@ -59,7 +59,9 @@ def make_bundle_version_info(versions):
         entry["feature_flags"] = [entry["flag"]]
         result[name] = entry
     dev = result["development"]
-    result["development"] = result[dev["pyodide_version"]] | dev
+    dev = result[dev["pyodide_version"]] | dev
```

Comment:
Pull out to separate PR

Reply:
https://github.com/cloudflare/workerd/pull/3991